tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/plugin.py

```diff
@@ -86,39 +89,38 @@ def _serve_index(self, request):
   def _serve_file(self, request):
     """Returns a resource file."""
     file_path = request.path
-    if file_path.startswith(plugin_folder):
-      file_path = file_path[len(plugin_folder):]
+    if file_path.startswith(_PLUGIN_DIRECTORY):
+      file_path = file_path[len(_PLUGIN_DIRECTORY):]
     res_path = os.path.join(os.path.dirname(__file__), file_path)
 
     with open(res_path, 'rb') as read_file:
       mimetype = mimetypes.guess_type(res_path)[0]
       return http_util.Respond(request, read_file.read(), content_type=mimetype)
 
   def is_active(self):
-    return bool(self._multiplexer.PluginRunToTagToContent(scalar_plugin_name))
+    return bool(self._multiplexer.PluginRunToTagToContent(_SCALAR_PLUGIN_NAME))
 
   def frontend_metadata(self):
     return base_plugin.FrontendMetadata(es_module_path="/static/index.js")
 
   def scalars_impl(self, tag, run):
-    """Result of the form `(body, mime_type)`."""
+    """Returns scalar data for the specified tag and run."""
```

Comment:
Can we use the proper pydoc by documenting `Raises:` and `Returns:`?

Reply:
Done.